curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/61539e07e7430b528fbffe21.md

```diff
@@ -7,17 +7,31 @@ dashedName: step-16
 
 # --description--
 
-Give your `#gallery` selector a `padding` property set to `0 4px`.
+Give your `.gallery` selector a `padding` property set to `20px 10px`.
```

Comment:
we need the reason, of why they are adding `padding` so they don't get lost while searching for it in the frame.

Reply:
I don't understand this comment?

Reply:
We are asking them to do something, and we aren't telling them what that something for, so they are left to guess why they did something, instead of how to implement something in the future

Reply:
I updated it but I will say I did it more to avoid the code block near punctuation. I think there must be plenty of steps in many challenges that are not explained in that way.

Reply:
I will create an issue for them, after this pr. 👍